configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
In file "wx/gtk/setup0.h" wxUSE_WXDIB is set to 1.
So, I am stopping work on this change till kosenko responds.

Tim S.